src/main/scala/pt/tecnico/dsi/openstack/designate/services/Zones.scala

```diff
@@ -0,0 +1,42 @@
+package pt.tecnico.dsi.openstack.designate.services
+
+import cats.effect.Sync
+import cats.syntax.flatMap._
+import fs2.Stream
+import org.http4s.client.Client
+import org.http4s.{Header, Query, Uri}
+import pt.tecnico.dsi.openstack.common.models.WithId
+import pt.tecnico.dsi.openstack.common.services.CrudService
+import pt.tecnico.dsi.openstack.designate.models._
+
+final class Zones[F[_]: Sync: Client](baseUri: Uri, authToken: Header)
+  extends CrudService[F, Zone, Zone.Create, Zone.Update](baseUri, "zone", authToken, wrapped = false) { self =>
+
+  def getByName(name: String, extraHeaders: Header*): F[WithId[Zone]] =
```

Comment:
Codacy found an issue: [Do not prefix getters with 'get'](https://app.codacy.com/gh/ist-dsi/scala-designate-client/pullRequest?prid=5821613)